rqd/tests/rqnimby_test.py

```diff
@@ -0,0 +1,143 @@
+# #!/usr/bin/env python
```

Comment:
Why this file rqd/tests/rqnimby_test.py is commented out?

Reply:
Merge issue. This has been fixed on the last merge from master.